server/Mac/mf_event.h

```diff
@@ -18,6 +18,7 @@
  */
 
 #ifndef __MF_EVENT_H
+#define __MF_EVENT_H
```

Comment:
Why didn't you remove the ```__``` prefix? As already pointed out these are reserved for compiler internal use.